orchestrator/careplanservice/service_test.go

```diff
@@ -1,17 +1,24 @@
 package careplanservice
 
 import (
-	"github.com/SanteonNL/orca/orchestrator/lib/auth"
-	"github.com/stretchr/testify/require"
+	"encoding/json"
+	"io"
 	"net/http"
 	"net/http/httptest"
 	"net/url"
+	"strings"
 	"testing"
+
+	"github.com/SanteonNL/orca/orchestrator/lib/auth"
+	"github.com/samply/golang-fhir-models/fhir-models/fhir"
+	"github.com/stretchr/testify/require"
 )
 
 var orcaPublicURL, _ = url.Parse("https://example.com/orca")
 var nutsPublicURL, _ = url.Parse("https://example.com/nuts")
 
+var taskJSON = "{\"resourceType\":\"Task\",\"id\":\"cps-task-01\",\"meta\":{\"versionId\":\"1\",\"profile\":[\"http://santeonnl.github.io/shared-care-planning/StructureDefinition/SCPTask\"]},\"text\":{\"status\":\"generated\",\"div\":\"<divxmlns=\\\"http://www.w3.org/1999/xhtml\\\"><pclass=\\\"res-header-id\\\"><b>GeneratedNarrative:Taskcps-task-01</b></p><aname=\\\"cps-task-01\\\"></a><aname=\\\"hccps-task-01\\\"></a><aname=\\\"cps-task-01-en-US\\\"></a><divstyle=\\\"display:inline-block;background-color:#d9e0e7;padding:6px;margin:4px;border:1pxsolid#8da1b4;border-radius:5px;line-height:60%\\\"><pstyle=\\\"margin-bottom:0px\\\">version:1</p><pstyle=\\\"margin-bottom:0px\\\">Profile:<ahref=\\\"StructureDefinition-SCPTask.html\\\">SharedCarePlanning:TaskProfile</a></p></div><p><b>status</b>:Requested</p><p><b>intent</b>:order</p><p><b>code</b>:<spantitle=\\\"Codes:{http://hl7.org/fhir/CodeSystem/task-codefullfill}\\\">fullfill</span></p><p><b>focus</b>:<ahref=\\\"Bundle-cps-bundle-01.html#urn-uuid-456\\\">Bundle:type=transaction</a></p><p><b>for</b>:Identifier:<code>http://fhir.nl/fhir/NamingSystem/bsn</code>/111222333</p><p><b>requester</b>:Identifier:<code>http://fhir.nl/fhir/NamingSystem/uzi</code>/UZI-1</p><p><b>owner</b>:Identifier:<code>http://fhir.nl/fhir/NamingSystem/ura</code>/URA-2</p><p><b>reasonReference</b>:<ahref=\\\"Bundle-cps-bundle-01.html#urn-uuid-789\\\">Bundle:type=transaction</a></p></div>\"},\"status\":\"requested\",\"intent\":\"order\",\"code\":{\"coding\":[{\"system\":\"http://hl7.org/fhir/CodeSystem/task-code\",\"code\":\"fullfill\"}]},\"focus\":{\"reference\":\"urn:uuid:456\"},\"for\":{\"identifier\":{\"system\":\"http://fhir.nl/fhir/NamingSystem/bsn\",\"value\":\"111222333\"}},\"requester\":{\"identifier\":{\"system\":\"http://fhir.nl/fhir/NamingSystem/uzi\",\"value\":\"UZI-1\"}},\"owner\":{\"identifier\":{\"system\":\"http://fhir.nl/fhir/NamingSystem/ura\",\"value\":\"URA-2\"}},\"reasonReference\":{\"reference\":\"urn:uuid:789\"}}"
```

Comment:
could use backticks instead of double quotes to avoid having to escape all the double qoutes to avoid readability. Or put it in a file?